kivy/uix/behaviors/focus.py

```diff
@@ -237,6 +237,15 @@ def _get_keyboard(self):
         2.0.0.
     '''
 
+    keyboard_suggestions = BooleanProperty(True)
+    '''If True provides auto suggestions on top of keyboard.
+    This will only work if :attr:`input_type` is set to `text`, `url`, `mail` or
+    `address`.
+    .. versionadded:: 2.1.0
```

Comment:
```suggestion

    .. versionadded:: 2.1.0

```

There needs to be blank lines around this, I believe.